src/App.css

```diff
@@ -19,3 +19,18 @@ body {
   justify-content: flex-end;
   flex-direction: column;
 } */
+
+.footer{
+  display: flex;
+  color: aliceblue;
+  align-items: baseline;
+  justify-content: center;
+  padding: 30px;
+  margin: 0 auto;
+  padding-top: 100px;
+  border-top: 2px solid white;
+  margin-top: 14.2%;
+  
```

Comment:
Works within the other elements on the page will continue to tweak styling of this footer with the other components this is a great start!